x-pack/test/case_api_integration/basic/tests/cases/find_cases.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+import { CASES_URL } from '../../../../../plugins/case/common/constants';
+import { postCaseReq, postCommentReq, findCasesResp } from '../../../common/lib/mock';
+import { deleteCases, deleteComments, deleteCasesUserActions } from '../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('legacyEs');
+  describe('find_cases', () => {
+    afterEach(async () => {
+      await deleteCases(es);
+      await deleteComments(es);
+      await deleteCasesUserActions(es);
+    });
+    it('should return empty response', async () => {
+      const { body } = await supertest
+        .get(`${CASES_URL}/_find`)
+        .set('kbn-xsrf', 'true')
+        .send()
+        .expect(200);
+
+      expect(body).to.eql(findCasesResp);
+    });
+
+    it('should return cases', async () => {
+      const { body: a } = await supertest
+        .post(CASES_URL)
+        .set('kbn-xsrf', 'true')
+        .send(postCaseReq);
+      const { body: b } = await supertest
+        .post(CASES_URL)
+        .set('kbn-xsrf', 'true')
+        .send(postCaseReq);
+      const { body: c } = await supertest
+        .post(CASES_URL)
+        .set('kbn-xsrf', 'true')
+        .send(postCaseReq);
+      const { body } = await supertest
+        .get(`${CASES_URL}/_find?sortOrder=asc`)
+        .set('kbn-xsrf', 'true')
+        .send()
+        .expect(200);
+
+      expect(body).to.eql({
+        ...findCasesResp,
+        total: 3,
+        cases: [a, b, c],
+        count_open_cases: 3,
+      });
+    });
+
+    it('filters by tags', async () => {
```

Comment:
Maybe we should at a test that filters by reporters or one that filters by both tags and reporters

Reply:
we need to figure out how to post from multiple users in the func tests to do this